cli_helpers/tabular_output/vertical_table_adapter.py

```diff
@@ -57,11 +57,7 @@ def vertical_table(data, headers, sep_title='{n}. row', sep_character='*',
 
     output = []
     for i, result in enumerate(formatted_rows):
-        output.append(_get_separator(i, sep_title, sep_character, sep_length))
-        output.append(result)
-        output.append('\n')
-
-    return ''.join(output)
+        yield _get_separator(i, sep_title, sep_character, sep_length) + result + '\n'
```

Comment:
@meeuw I don't think we need the newline character any longer since we're returning each record separately. It's the client code's job to insert a newline between each record. Then you can remove the extra blank line you had to add to the tests.